README.md

```diff
@@ -1,2 +1,17 @@
 # metal-key
+
 Adds a custom event that helps listen to key events in a cleaner way
+
+## Setup
+
+`npm install --save-dev metal-key`
+
+## How to use
+
+Lets say you want to listen to the `Enter` key and execute a function once it happens.
+
+`dom.on('keydown:enter', function() { console.log('Enter key was pressed') })`.
```

Comment:
Nice, thanks for adding a usage explanation!

I was just thinking, it might be good to have it be clear that you need to import `metal-key` in your file for this to work, even though you may not need its exported value. Just adding the import statements here would help, for example:

```js
import dom from 'metal-dom';
import 'metal-key'; // Needs to be imported for the custom event to work.

dom.on('keydown:enter', function() { console.log('Enter key was pressed') })
```